src/box/errcode.h

```diff
@@ -141,287 +141,313 @@ struct errcode_record {
+	_(ER_NO_SUCH_SESSION, 254,		"Session %llu does not exist", "sid", ULLONG) \
+	_(ER_WRONG_SESSION_TYPE, 255,		"Session '%s' is not supported", "session_type", STRING) \
 	_(ER_PASSWORD_EXPIRED, 256,		"Password expired") \
 	_(ER_AUTH_DELAY, 257,			"Too many authentication attempts") \
 	_(ER_AUTH_REQUIRED, 258,		"Authentication required") \
-	_(ER_SQL_SEQ_SCAN, 259,			"Scanning is not allowed for %s") \
-	_(ER_NO_SUCH_EVENT, 260,		"Unknown event %s") \
-	_(ER_BOOTSTRAP_NOT_UNANIMOUS, 261,	"Replica %s chose a different bootstrap leader %s") \
-	_(ER_CANT_CHECK_BOOTSTRAP_LEADER, 262,	"Can't check who replica %s chose its bootstrap leader") \
+	_(ER_SQL_SEQ_SCAN, 259,			"Scanning is not allowed for %s", "object", STRING) \
+	_(ER_NO_SUCH_EVENT, 260,		"Unknown event %s", "even", STRING) \
+	_(ER_BOOTSTRAP_NOT_UNANIMOUS, 261,	"Replica %s chose a different bootstrap leader %s", "replica", STRING, "bootstrap_leader", STRING) \
+	_(ER_CANT_CHECK_BOOTSTRAP_LEADER, 262,	"Can't check who replica %s chose its bootstrap leader", "replica", STRING) \
 	_(ER_BOOTSTRAP_CONNECTION_NOT_TO_ALL, 263, "Some replica set members were not specified in box.cfg.replication") \
 	_(ER_NIL_UUID, 264,			"Nil UUID is reserved and can't be used in replication") \
 	_(ER_WRONG_FUNCTION_OPTIONS, 265,	"Wrong function options: %s") \
 	_(ER_MISSING_SYSTEM_SPACES, 266,	"Snapshot has no system spaces") \
-	_(ER_CLUSTER_NAME_MISMATCH, 267,	"Cluster name mismatch: name '%s' provided in config confilcts with the instance one '%s'") \
-	_(ER_REPLICASET_NAME_MISMATCH, 268,	"Replicaset name mismatch: name '%s' provided in config confilcts with the instance one '%s'") \
-	_(ER_INSTANCE_NAME_DUPLICATE, 269,	"Duplicate replica name %s, already occupied by %s") \
-	_(ER_INSTANCE_NAME_MISMATCH, 270,	"Instance name mismatch: name '%s' provided in config confilcts with the instance one '%s'") \
-	_(ER_SCHEMA_NEEDS_UPGRADE, 271,		"Your schema version is %u.%u.%u while Tarantool %s requires a more recent schema version. Please, consider using box.schema.upgrade().") \
+	_(ER_CLUSTER_NAME_MISMATCH, 267,	"Cluster name mismatch: name '%s' provided in config confilcts with the instance one '%s'", \
+						"actual", STRING, "expected", STRING) \
+	_(ER_REPLICASET_NAME_MISMATCH, 268,	"Replicaset name mismatch: name '%s' provided in config confilcts with the instance one '%s'", \
+						"actual", STRING, "expected", STRING) \
+	_(ER_INSTANCE_NAME_DUPLICATE, 269,	"Duplicate replica name %s, already occupied by %s", "name", STRING, "uuid", STRING) \
+	_(ER_INSTANCE_NAME_MISMATCH, 270,	"Instance name mismatch: name '%s' provided in config confilcts with the instance one '%s'", \
+						"actual", STRING, "expected", STRING) \
+	_(ER_SCHEMA_NEEDS_UPGRADE, 271,		"Your schema version is %s while Tarantool %s requires a more recent schema version. " \
+						"Please, consider using box.schema.upgrade().", \
+						"schema_version", STRING, "tarantool_version", STRING) \
```

Comment:
"actual", "expected"

Reply:
Discussed f2f and decided do keep it this way. Strictly speaking `tarantool_version` could not be called expected.